Validation/CaloTowers/interface/CaloTowersValidation.h

```diff
@@ -34,9 +34,9 @@ class CaloTowersValidation : public DQMEDAnalyzer {
  public:
    CaloTowersValidation(edm::ParameterSet const& conf);
   ~CaloTowersValidation();
-  virtual void analyze(edm::Event const& e, edm::EventSetup const& c);
-  virtual void beginRun() ;
-  virtual void endRun() ;
+  virtual void analyze(edm::Event const& e, edm::EventSetup const& c) override;
+  virtual void beginRun();
```

Comment:
The framework will never call these functions (`beginRun` and `endRun`) since they have the wrong argument list. It looks like there is a bug in this analyzer (but shouldn't be fixed in this pull request).